Include/internal/pycore_object.h

```diff
@@ -29,6 +29,8 @@ _PyType_HasFeature(PyTypeObject *type, unsigned long feature) {
 
 extern void _PyType_InitCache(PyInterpreterState *interp);
 
+/* Only private in Python 3.10 and 3.9.8+; public in 3.11 */
+extern PyObject *_PyType_GetQualName(PyTypeObject *type);
```

Comment:
oh, nice solution! thank you for sorting it out.

Reply:
To quote Victor Stinner:

> Extern ensures that GCC/clang which supports `-fvisibility` will not export the symbol